### PR TITLE
[Running_Profile] Fix all counter in DataStreamRecv and change the image path in docs

### PR DESCRIPTION
**Pull request by HappenLee**
RT

### CODE REVIEW

**Review by morningman — DISMISSED** [review]
LGTM

File: docs/zh-CN/administrator-guide/running-profile.md [hunk 1]
```diff
@@ -92,18 +92,23 @@ BE端收集的统计信息较多，下面列出了各个参数的对应含义：
    - AverageThreadTokens: 执行Fragment使用线程数目，不包含线程池的使用情况
    - Buffer Pool PeakReservation: Buffer Pool使用的内存的峰值
    - MemoryLimit: 查询时的内存限制
-   - PeakMemoryUsage: 内存使用的峰值  
+   - PeakMemoryUsage: 整个Fragment在查询时内存使用的峰值
```
Comment (wutiangan) [inline docs/zh-CN/administrator-guide/running-profile.md:95]: Fragment -> 1个instance

File: docs/zh-CN/administrator-guide/running-profile.md [hunk 2]
```diff
@@ -92,18 +92,23 @@ BE端收集的统计信息较多，下面列出了各个参数的对应含义：
...
   - MaxBlockSize: 单个Block的大小
   - TotalReadBlockTime: 读Block的总耗时
 
+#### `DataStreamRecvr`
+  - BytesReceived: 通过网络接收的数据量大小
+  - DataArrivalWaitTime: 等待Sender发送数据的总时间
+  - FirstBatchArrivalWaitTime: 等待第一个batch从Sender获取的时间
+  - DeserializeRowBatchTimer: 反序列化网络数据的耗时
+  - SendersBlockedTotalTimer(*): DataStreamRecv的队列的内存被打满，Sender端等待的耗时
+
```
Comment (wutiangan) [inline docs/zh-CN/administrator-guide/running-profile.md:111]: DataStreamRecv放在exchangeNode里面

Comment (HappenLee) [reply docs/zh-CN/administrator-guide/running-profile.md:111]: ok

File: docs/zh-CN/administrator-guide/running-profile.md (hunk 1 above)
Comment (HappenLee) [reply docs/zh-CN/administrator-guide/running-profile.md:95]: ok

**Review by morningman — DISMISSED** [review]
Approve

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**